src/managers/queue-manager.js

```diff
@@ -19,20 +17,13 @@ export async function startQueueListener() {
     if (getUserToken()) {
       log("Queue watcher started");
       pollingSetup = true;
-      poll(() => new Promise(() => pollMessageQueue()), 60000);
+      poll(() => new Promise(() => pollMessageQueue()), 10000);
```

Comment:
Maybe we can define a pair of useful constants here and use seconds as the unit of the polling delay? 

// Use seconds directly 
const MILISECONDS_IN_A_SECOND = 1000;
const POLLING_DELAY_SECONDS = 10;


```suggestion
...
      poll(() => new Promise(() => pollMessageQueue()), POLLING_DELAY_SECONDS * MILISECONDS_IN_A_SECOND);
...
```
or maybe a bit simpler:

```suggestion
...
      poll(() => new Promise(() => pollMessageQueue()), POLLING_DELAY_SECONDS * 1000);
...
```

Reply:
`const POLLING_DELAY_IN_SECONDS = 1000 * 10` is another way.